src/python/fuzzing/tests.py

```diff
@@ -823,10 +822,6 @@ def get_command_line_for_application(file_to_run='',
     return android.adb.get_application_launch_command(
         all_app_args, testcase_path, testcase_file_url)
 
-  elif plt == 'FUCHSIA' and not launcher:
```

Comment:
note: this stuff may be needed if you ever want to support black box fuzzing on fuchsia.

Reply:
so may be worth replacing these with a TODO instead :)